python/katana_setup.py

```diff
@@ -349,9 +350,22 @@ def _build_cython_extensions(pyx_files, source_root_name, extension_options):
 def cythonize(module_list, *, source_root, **kwargs):
     # TODO(amp): Dependencies are yet again repeated here. This needs to come from a central deps list.
     require_python_module("packaging")
-    require_python_module("Cython", "0.29.12")
     require_python_module("numpy", "1.10")
-    require_python_module("pyarrow", "1.0", "3.0.dev")
+    try:
+        require_python_module("Cython", "0.29.12")
+        require_python_module("pyarrow", "4.0", "5.0.dev")
+        build_extensions = True
+    except RequirementError:
+        print(
+            "WARNING: Building Katana Python without extensions! The following features will not work: katana.local, "
+            "katana.distributed, katana parallel loops. The following features will work: katana_enterprise.client and "
```

Comment:
katana_enterprise keeps appearing at more places than desired. As planned, it would be nice to hide katana_enterprise behind katana package

Reply:
I included it here because many people are still not aware of the hiding of it and this warning will be temporary. But you are right. I will change it.